util/test/checkChplInstall

```diff
@@ -134,12 +150,12 @@ fi
 
 # Check for valid launchers
 if [ ${chpl_launcher} == "slurm-srun" -o ${chpl_launcher} == "amudprun" -o ${chpl_launcher} == "none" ]; then
-    echo "\$CHPL_LAUNCHER=${chpl_launcher} is compatible with test script."
+    log_info "\$CHPL_LAUNCHER=${chpl_launcher} is compatible with test script."
 else
-    echo ""
-    echo "\$CHPL_LAUNCHER=${chpl_launcher} is not compatible with test script."
-    echo "This does not necessarily indicate that your Chapel installation is incorrect."
-    echo "See \$CHPL_HOME/doc/launcher.rst for information on manually launching Chapel programs."
+    log_info ""
```

Comment:
The line will look odd now that it'll have some text in instead of just being a new line. Maybe just delete this?